cmd/jaeger/internal/integration/integration.go

```diff
@@ -38,7 +41,9 @@ type E2EStorageIntegration struct {
 // This function should be called before any of the tests start.
 func (s *E2EStorageIntegration) e2eInitialize(t *testing.T) {
 	logger, _ := testutils.NewLogger()
+	s.ConfigFile = createStorageCleanerConfig(t, s.ConfigFile)
 
+	fmt.Println(s.ConfigFile)
 	cmd := exec.Cmd{
 		Path: "./cmd/jaeger/jaeger",
 		Args: []string{"jaeger", "--config", s.ConfigFile},
```

Comment:
curious, what does the first argument "jaeger" refer to?

Reply:
Args slice includes the actual command itself as the first element `(Args[0])`.